### PR TITLE
Experiment on issue 953 data race

### PR DESCRIPTION
**Pull request by lemire**

### CODE REVIEW

**Comment by lemire** [comment]
@jkeiser  See? All better. This validates (but does not prove) my theory that the custom command thing creates a data race.

File: singleheader/CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ set(SINGLEHEADER_FILES
   ${CMAKE_CURRENT_BINARY_DIR}/simdjson.h
   ${CMAKE_CURRENT_BINARY_DIR}/amalgamate_demo.cpp
   ${CMAKE_CURRENT_BINARY_DIR}/README.md
+  ${CMAKE_CURRENT_SOURCE_DIR}/done.txt
```
Comment (jkeiser) [inline singleheader/CMakeLists.txt:10]: Should be CMAKE_CURRENT_BINARY_DIR, I think. I'll integrate this into my PR ...

Comment (jkeiser) [reply singleheader/CMakeLists.txt:10]: I'm going to see what happens if I don't include done.txt. Theoretically at least, the other files should always get updated.

**Comment by lemire** [comment]
@jkeiser I'll close this PR. It was never meant to be merged, just illustrative.

File: singleheader/CMakeLists.txt (hunk 1 above)
Comment (lemire) [reply singleheader/CMakeLists.txt:10]: Yes. But you have to respect the rule that only one target should directly depend on the custom command. If you do so, it should work.

**Comment by jkeiser** [comment]
@lemire confusingly, `make -j` on my machine continues to repro the issue even with this patch ... but I think I'm going to try something like it and see if it at least makes CI better ...

**Comment by jkeiser** [comment]
@lemire oh, even weirder. I'm getting something like the repro no matter what, with this branch. Must be something I ate.

**Pull request closed without merging**